### ISSUE TITLE
The .exe application is not opening

### ISSUE THREAD

**Issue by TheHugeFluff** [KuhakuPixel/AjaxChessBot#3]
I don't know if it works now currently or not but when ever I open the .exe it doesnt do anything. I can provide a video if needed. I found out the .exe opens the command prompt for a split second then nothing happens.

**Comment by KuhakuPixel** [comment]
@TheHugeFluff Yeah providing a video would be great, you can share the link of the uploaded video

**Comment by TheHugeFluff** [comment]
Shit, sorry for the late reply but yea I'll have to reinstall but yea I'll do the video. How shall I record or well post the video on?

**Comment by KuhakuPixel** [comment]
If you are on windows you can use [obs_Studio](https://obsproject.com/) which is free and opensource, to record the video
then you can upload the video to cloud storage like [google drive](https://drive.google.com/) then share the link of the video here
👍

**Comment by TheHugeFluff** [comment]
https://drive.google.com/file/d/1A8UNjiZtXd5ICSOnxyB5p9rH6s0_AE-F/view?usp=sharing

**Comment by KuhakuPixel** [comment]
@TheHugeFluff thanks for the detailed report, I will try to find  possible solutions/workaround soon

**Comment by KuhakuPixel** [comment]
@TheHugeFluff 

Here are the possible solutions I can think of right now:

1. You don't have `.NET runtime.` which is required  to run a c# based application
You can download it [here](https://dotnet.microsoft.com/en-us/download/dotnet/6.0/runtime) 
Choose the download for windows and choose `x86` if you are on 32 bit system, choose `x64` if you are
on 64 bit system. Run the ***installer*** then install it.

2. Check  your antivirus, if it is blocking the program since this program uses `win api`  that makes a mouse click to move the `chess pieces`. So the antivirus might see it as a threat,

**Comment by KuhakuPixel** [comment]
@TheHugeFluff if this solves the issue let me know

**Comment by TheHugeFluff** [comment]
Still not working
btw i recorded proof

**Comment by KuhakuPixel** [comment]
if you are willing we can hop into discord to find possible solutions

my id is `Nicholaspixel#2786`

**Comment by natepeter** [comment]
@KuhakuPixel can U help me
I added you on discord

**Comment by KuhakuPixel** [comment]
@natepeter can you add me again? , I cancelled by mistake

**Comment by ChessGrandMasterLouis** [comment]
The .exe file still dont want to open!

**Comment by KuhakuPixel** [comment]
> The .exe file still dont want to open!

Try to compile the exe from source

**Comment by ChessGrandMasterLouis** [comment]
Where is the "source, so i can compile it.

**Comment by ChessGrandMasterLouis** [comment]
source*

**Comment by KuhakuPixel** [comment]
> Where is the "source, so i can compile it.

its here https://github.com/KuhakuPixel/AjaxChessBot
you git clone it, download a compiler and compile it to produce an exe

**Comment by SomeRandomBoi** [comment]
Im having the same issues as @TheHugeFluff have yall found out the problem?

**Comment by SomeRandomBoi** [comment]
Wait I found out how
Just open it using command prompt and pasting the exe file path into it
Hope this helps!

**Comment by nacremo** [comment]
@SomeRandomBoi  can you take a video for the solve of this problem? please.

**Comment by SomeRandomBoi** [comment]
I can't take a video but here's the basic walkthrough.
Press Win+R to bring up cmd, then type C:
cd C:\Downloads
(example).exe 
and it should work @nacremo

**Comment by mocks4L** [comment]
install this .net framework : 

> https://download.visualstudio.microsoft.com/download/pr/476eba79-f17f-49c8-a213-0f24a22cd026/37c02de81ff5b76ac57a5427462395f1/dotnet-runtime-3.1.32-win-x64.exe

**Comment by FiyinOjo637** [comment]
@mocks4L  Thanks so much. All those who have problem should just download the link. adios

**Comment by prosto4lik** [comment]
Hello, the exe has opened a problem with stockfish . I opened it, but the import didn't fit in.What to do?
